docs/source/python/getting_involved.rst

```diff
@@ -33,3 +33,42 @@ integrations in other projects, we'd be happy to have you involved:
  * Follow our activity on `JIRA <https://issues.apache.org/jira/browse/ARROW>`_
  * Learn the `Format / Specification
    <https://github.com/apache/arrow/tree/master/format>`_
```

Comment:
Could we correct the indentation in the lines before the Architectural overview?

Reply:
done!